core/src/main/scala/org/apache/spark/sql/crossdata/security/Credentials.scala

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2015 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.crossdata.security
+
+case class Credentials(user: Option[String] = None, password: Option[String] = None, sessionId: Option[String] = None)
```

Comment:
Maybe, at least, user or sessionId must exists, because there is not credentials without them, isn't it?